src/cascadia/Remoting/WindowManager.cpp

```
@@ -118,78 +118,60 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
-                                      "WindowManager_proposeToMonarch_kingDied",
-                                      TraceLoggingLevel(WINEVENT_LEVEL_VERBOSE),
-                                      TraceLoggingKeyword(TIL_KEYWORD_TRACE));
-
-                    // We failed to ask the monarch. It must have died. Try and
-                    // find the real monarch. Don't perform an election, that
-                    // assumes we have a peasant, which we don't yet.
-                    _createMonarchAndCallbacks();
-                    // _createMonarchAndCallbacks will initialize _isKing
-                    if (_isKing)
-                    {
-                        // We became the king. We don't need to ProposeCommandline to ourself, we're just
-                        // going to do it.
-                        //
-                        // Return early, because there's nothing else for us to do here.
-                        TraceLoggingWrite(g_hRemotingProvider,
-                                          "WindowManager_proposeToMonarch_becameKing",
-                                          TraceLoggingLevel(WINEVENT_LEVEL_VERBOSE),
-                                          TraceLoggingKeyword(TIL_KEYWORD_TRACE));
+                //  They hopefully just died here. That's okay, let's just go
+                // ask the next in the line of succession. At the very worst,
+                // we'll find _us_, (likely last in the line).
 
-                        // In WindowManager::ProposeCommandline, had we been the
-                        // king originally, we would have started by setting
-                        // this to true. We became the monarch here, so set it
-                        // here as well.
-                        _shouldCreateWindow = true;
-                        return;
-                    }
+                // If the monarch (maybe us) failed for _any other reason_ than

```

Also, this comment doesn't seem to follow from the previous paragraph. We're eating all the exceptions, not bubbling any exceptions out!

---

whoops, missed one.